### ISSUE TITLE
Result of graph.users.paged() does not have property of 'value'.

### ISSUE THREAD

**Issue by fischkoder** [pnp/pnpjs#2833]
### What version of PnPjs library you are using

3.x

### Minor Version Number

3.19.0

### Target environment

SharePoint Framework

### Additional environment details

Sharepoint web part, get authentication from context.

### Question/Request

I am working on get the paging result by using graph.users.top().paged().

Based on the sample on the pnp.js/graph document, I saw the result can be easily accessed by .value .
but I have check the return type of .paged(), it's a asyncIterable<getType> which does not have any property to get value directly.
Is there any change in paged()? or Is the sample on the document just a pseudo code?

Thanks.

**Comment by bcameron1231** [comment]
HI. It looks like the documentation is just inaccurately providing the way to use Async Iterators. As it's returning an Async Iterator, you should use the for await...of loop. 

```
 for await (const value of await graph.users.top(10).paged()) {
      console.log(value);
  }
```

We'll get this sorted. Thanks!

**Comment by bcameron1231** [comment]
Hi. It looks like the documentation is may be inaccurately providing the way to use paging with Async Iterators. As it's returning an Async Iterator, you should use the for await...of loop. 

```
 for await (const value of await graph.users.top(10).paged()) {
      console.log(value);
  }
```

We'll take a look further and update accordingly.

**Comment by bcameron1231** [comment]
We're going to push a hotfix out to fix this. Apologies for the inconvenience.

**Comment by bcameron1231** [comment]
Fix is in latest release 3.20.1
